main.nf

```diff
@@ -629,10 +630,15 @@ process Normalize_DWI {
 
     output:
     set sid, "${sid}__dwi_normalized.nii.gz" into dwi_for_resample
+    file "${sid}_fa_wm_mask.nii.gz"
 
     script:
     """
-    dwinormalise $dwi $mask ${sid}__dwi_normalized.nii.gz -fslgrad $bvec $bval
+    scil_compute_dti_metrics.py $dwi $bval $bvec --mask $mask\
```

Comment:
Shouldn't you do it only on the DTI shells?

Reply:
You're right ! I will fix that

Reply:
I currently run a test